### ISSUE TITLE
Research needed for: amenity/bank|Commerce Bank

### ISSUE THREAD

**Issue by bhousel** [osmlab/name-suggestion-index#243]
#### :thinking: We need help deciding what to do with "amenity/bank|Commerce Bank"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `amenity/bank|Commerce Bank` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `amenity/bank|Commerce Bank`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `amenity/bank|Commerce Bank`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

**Comment by ENT8R** [comment]
Isn't this just this bank: https://www.commercebank.com/ ?
When looking at Overpass Turbo, it is also located only in the USA: https://overpass-turbo.eu/s/Hc2

**Comment by Adamant36** [comment]
It might be just that bank currently tagged. For it was a question of if we wanted to add commerce banks in other countries with their own entries/country codes or not. I probably should have commented as much, but I totally spaced it.

**Comment by bhousel** [comment]
FYI to whomever is watching this repository.. I'm going to close all of these old "Research Needed" tickets.

I added these back in October to give Hacktoberfest volunteers something to do, and it was a great success!  Back then, we had just processed the OSM planet for the first time in many years, were just starting to match brands to Wikidata qids, and we were able to match thousands of brands very quickly with a lot of help 💪 

Today, we have a great core group of maintainers regularly processing updates from OSM, we've added many new names and categories, changed the instructions a few times, and it's easy to just look through https://osmlab.github.io/name-suggestion-index/brands/ for things to do, rather than having these old tickets hanging around.  

A lot has changed in 9 months, so bye bye old tickets! 🔪
